src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js

```diff
@@ -14,9 +14,19 @@ import { forwardRef, useEffect } from '@wordpress/element';
 import { __unstableUseNavigateRegions as useNavigateRegions } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { useMergeRefs } from '@wordpress/compose';
+import bgAiImg from '../../static/images/ai_bg.jpg';
```

Comment:
Can't we use the same way of mentioning in `_icons` and not directly reference a static file?